upstairs/src/lib.rs

```diff
@@ -1162,28 +1178,30 @@ where
      * also request extent versions and will have to repair this
      * downstairs, skipping over step 3 as well.
      *
-     * (Offline only):
-     *
+     * NegotiationState::GetLastFlush (offline only)
```

Comment:
The comment above here should be updated to replace `3` and `4` with 
`NegotiationState::GetLastFlush` and `NegotiationState::GetExtentVersions`

Reply:
Done in 97dfe4778518d3536ecc2adb75bf2b9672296416